src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/RazorLanguageServerClient.cs

```diff
@@ -49,7 +49,11 @@ internal class RazorLanguageServerClient : ILanguageClient, ILanguageClientCusto
         private const string RazorLSPLogLevel = "RAZOR_TRACE";
 
         public event AsyncEventHandler<EventArgs>? StartAsync;
-        public event AsyncEventHandler<EventArgs>? StopAsync;
+        public event AsyncEventHandler<EventArgs>? StopAsync
+        {
+            add { }
```

Comment:
Think I left a similar comment in another PR asking if this was needed and if so whether we also needed to define `StartAsync`

Reply:
Oh I think I misunderstood the other comment and thought I'd resolved it, sorry!

`StartAsync` and `StopAsync` are part of `ILanguageClient`, Start is used in our code, but Stop is not, causing analyzers to complain about an un-used member. This can be avoided with the add/remove parts, [which is what we do today](https://github.com/dotnet/razor-tooling/blob/main/src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/RazorLanguageServerClient.cs#L118-L123).